app.rb

```diff
@@ -0,0 +1,78 @@
+# 	@types = ["Normal", "Fire", "Water", "Electric", "Grass", "Ice", "Fighting", "Poison", "Ground", "Flying", "Psychic", "Bug", "Rock", "Ghost", "Dragon", "Dark", "Steel", "Fairy"]
+# 	@tenant = tenant.find(params[:id]);
+# 	erb :"tenant/edit"
+# end
```

Comment:
are you tenants pokemon? haha. Yea, so for these routes you'd need to include apartments ..so like '/apartments/:id/tenants' for index.